SCRIPT.md

```diff
@@ -68,13 +68,13 @@ We will look at ways to avoid this.
 
 ## Theory: What's certificates and Public Key Infrastructure really about?
 
-The trust estabilished in a Public Key Infrastructure is based on a few base assumptions:
+The trust established in a Public Key Infrastructure is based on a few base assumptions:
 
 * There exists keypairs with the mathematical property that what is encrypted by one of the members of the pair
   can be decrypted with the other and vice-versa, while knowing one doesn't make it possible to uncover the other.
   A metaphor of padlocks and keys can be imagined.
 * Each party can freely share their public key but must guard their private key
-* Since the relationship between the keys is symmetric, this can be used both for encryption and for authentication.
+* Since the relationship between the keys is asymmetric, this can be used both for encryption and for authentication.
```

Comment:
The *relationship* is symmetric: What is encoded with public can be decoded with private *and vice versa* :-)

All the other changes I agree with :-)

Reply:
There is one place where the change is incorrect. If you agree and update, I'll merge :-)

Reply:
I see. I reverted the change now.